client/src/main.h

```diff
@@ -119,6 +119,7 @@ struct AppState
   uint64_t              lastRenderTime;
   bool                  lastRenderTimeValid;
   RingBuffer            renderTimings;
+  RingBuffer            renderDuration;
```

Comment:
Perhaps these should all be renamed to something more accurate, ie:
```
uploadTiming
renderTiming
renderDuration
```

Reply:
I renamed `frameTimings` to `uploadTimings`. Not sure how the others would be renamed. I think `renderTimings` and `renderDuration` makes sense.